conf/application.conf

```diff
@@ -18,6 +18,7 @@ include "backend.conf"
 # ~~~~~
 
 appName = cip-phone-number-verification
+appUrl = "https://"${appName}".protected.mdtp"
```

Comment:
I don't see this being used anywhere, is it needed?

Reply:
removed